apps/web/src/auth.ts

```diff
@@ -86,29 +86,30 @@ const providers: Provider[] = [
       return generateOtp();
     },
     async sendVerificationRequest({ identifier: email, token, url }) {
-      const user = await prisma.user.findUnique({
-        where: {
-          email,
-        },
-        select: {
-          name: true,
-          locale: true,
-        },
-      });
+      try {
```

Comment:
I think this changes in this file should be reverted because:

* There's no point in adding a `try..catch` if we are throwing the error in the catch statement.
* Sending a registration email from the login form won't allow the user to register a new account. It can only be done from the registration form.